script/Arcueid/Day 5/05_00_ARC05_1B.txt

```diff
@@ -376,7 +377,7 @@ But, no, that's...
 [sha:14e4eb6110a944f9f5a54d29f94cbbc414c3357c]{
 -- Page 64, Offset 8748.
 -- 志貴が今まで何人殺してきたかは知らないけど、獲物にわたしを選んだ時点で年貢の納め時だったんじゃない？」
-I don't know how many people you've killed until now, but it must have been time for you to pay the price when you chose me as your next prey."
+I don't know how many people you've killed until now, but I think the moment you chose me as your next prey, the time to pay the price had come."
```

Comment:
Might emphasize that _he_ is paying the price
> ...know how many people you've killed so far, but I think when you selected me as your prey, it meant it was finally time for you to pay the piper.

Reply:
Pay the piper is the correct equivalent. They're almost identical in meaning. Sometimes translation treats you kindly.